openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -98,6 +97,8 @@ class AccountView(APIView):
             * year_of_birth: The year the user was born, as an integer, or null.
             * account_privacy: The user's setting for sharing her personal
               profile. Possible values are "all_users" or "private".
+            * has_accomplishments: Signals whether badges are enabled on the
```

Comment:
This is causing problems. I shall change this field to read 'accomplishments_shared' so that it is more semantically correct. Setting this false for an empty accomplishment set, going way out of our way to load the accomplishments tab, inject an empty collection, and try hoping that no Backbone event attempts to reload the collection is hacky at best. This indicator should be for whether or not the accomplishment tab is loaded, not whether or not they have badges, for the sake of maintainable code.